src/main/java/seedu/address/commons/util/CollectionUtil.java

```diff
@@ -1,11 +1,10 @@
 package seedu.address.commons.util;
 
+import seedu.address.model.location.Location;
+
 import static java.util.Objects.requireNonNull;
 
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Objects;
+import java.util.*;
```

Comment:
I believe we can turn off auto-import on Intellij. It is generally not encouraged to mass-import unnecessary items.

Reply:
This was already fixed in a previous commit.